environments/stg/backend_lb_config.yaml

```diff
@@ -370,6 +370,8 @@ gateways:
         component: case-api
       - product: sptribs
```

Comment:
As above, can we remove dss-backend?

Reply:
we will do that by separate PR as we agreed